### ISSUE TITLE
Define zine data format

### ISSUE THREAD

**Issue by wayspurrchen** [RobotArmsClub/RobotArmsClub#1]
We need to come up with a data format for the Zine to use. I have a couple thoughts up front:
- I'd like it to be a node module since the NPM tooling is pretty good but that doesn't mean that the file contents literally need to be consumable JavaScript - I just want to be able to clone ti down.
- We should segment each "page" of the zine into its own file and concatenate them later, so that it's easy to deal with pagination in different formats.

Besides that, carpe diem. @flipactual I'm assigning this to you since I know that you have some thoughts re: how to handle a markdown + React input.

**Comment by flipactual** [comment]
Why would we want to publish content on npm? I'm not understanding any benefit there over publishing it on GitHub.

What do you mean by diving up pages into files? It makes sense to me to have one file per entry, but I'm not sure how or why we'd divide entries.

**Comment by wayspurrchen** [comment]
@flipactual You're right, let's publish it on GitHub. I was thinking of my old approach of putting NPM packages into bitbucket and installing the dependencies directly from the bitbucket URL, but there's no reason we can't do the exact same thing on GitHub.

I guess it doesn't actually matter if we divide the entries into separate files. The important piece to me is that there's a demarcation between what would appear to be a page in the different formats, so it's clear where to "break" in its different representations. Does that make sense?

**Comment by flipactual** [comment]
To be frank (and/or realistic) there may be some limitations/concessions for a while with regard to formatting. I'm all for embracing those limitations, but, of course, also open to hearing about issues others have with them.

I will ensure that one article doesn't start on the same page as another begins though. Is that sufficientish?

**Comment by wayspurrchen** [comment]
Yup, that's totally good with me! There's lots of room for iteration / experimentation, as far as I'm concerned. 👍

**Comment by flipactual** [comment]
Multiple parties want this from me, so I've put it on my list for 2016/04/27 😎

I will be taking an app approach as opposed to module approach, but I am of the opinion that the content should be maintained in a separate repository.

We'll run the app with something like `npm run compile ../rac-content/2016/06/` and it'll look for a manifest in that directory that tells it what to do from there.

**Comment by beaugunderson** [comment]
One way to do Markdown-based format would be to use [remark](https://github.com/wooorm/remark) and implement formatting as remark plugins... (responding to messages in the Slack re: Markdown)

**Comment by flipactual** [comment]
@beaugunderson I like @wooorm so much I'm inclined to switch to that from https://www.npmjs.com/package/marked

**Comment by flipactual** [comment]
re PDF and possibly other things soon: https://github.com/RobotArmsClub/rac-pub
